readium/navigator/src/main/java/org/readium/r2/navigator/epub/EpubNavigatorFactory.kt

```diff
@@ -47,6 +52,7 @@ class EpubNavigatorFactory(
             publication = publication,
             baseUrl = null,
             initialLocator = initialLocator,
+            readingOrder = readingOrder ?: publication.readingOrder,
```

Comment:
```suggestion
            readingOrder = readingOrder
```
If `readingOrder` is never null, positions will always be used in the fragment.